### ISSUE TITLE
EasyTouch + Intelliflow + Intellichlor + Ultratemp Hybrid ETI doesn't show up as controller properly in dash panel

### ISSUE THREAD

**Issue by terafin** [tagyoureit/nodejs-poolController#389]
**Describe the bug**
This has been happening for a while, and I've been ignoring it - but now that I have the ETI I thought I'd file! 

TLDR; My poolcontroller/dashpanel docker container is configured via docker-compose on a raspi 3b. Things seem to work, but lots of shows up as Error 404 whenever I access parts of the UI

**To Reproduce**
1. Go to 'Settings -> Controller'
2. See error 404

**Expected behavior**
Shows my controller config

**Screenshots**
Attached

**Packet Capture**
Attached
**Pool Equipment**
 - Controller: EasyTouch 8
 - Controller Firmware Version: 2.180
 - Pump(s) manufacturer and model: IntelliFlow 2
 - Chlorinator: IntelliChlor-40
 - Heater(s): Ultratemp ETI Hybrid
 - Chemical controller: None
 - Valves: None
 - Any other relevant equipment: None

**Desktop (please complete the following information):**
 - OS: macOS 13 and Windows 11
 - 
<img width="1552" alt="Screen Shot 2022-02-24 at 11 11 21 PM" src="https://user-images.githubusercontent.com/66893/155671313-242dac6e-19cd-463e-8862-113912900483.png">
<img width="1552" alt="Screen Shot 2022-02-24 at 11 11 17 PM" src="https://user-images.githubusercontent.com/66893/155671329-a6bd61ce-ac5e-46af-894f-557836c5577f.png">
<img width="1552" alt="Screen Shot 2022-02-24 at 11 11 12 PM" src="https://user-images.githubusercontent.com/66893/155671348-20cbfe55-08fa-42ed-b522-e2bc25f5fbd7.png">
Browser: Edge, Chrome, Safari

**Comment by terafin** [comment]
[replay.zip](https://github.com/tagyoureit/nodejs-poolController/files/8139241/replay.zip)

**Comment by terafin** [comment]
Closing this, sorry for the noise. Turns out, you need to pass in TZ in docker for this to work :) Timezones were off!

eg:
```
    environment:
      TZ: America/Los_Angeles
```
